.github/workflows/build_and_test_cosmian_vm.yml

```diff
@@ -0,0 +1,174 @@
+        uses: "google-github-actions/auth@v2"
+        with:
+          credentials_json: "${{ secrets.GOOGLE_COSMIAN_DEV_CREDENTIALS }}"
+
+      - name: Build GCP images - main
+        env:
+          TIMESTAMP: ${{ steps.env.outputs.TIMESTAMP }}
+        if: ${{ !startsWith(github.ref, 'refs/tags') }}
+        run: |
+          packer build -var "prefix=alpha-$TIMESTAMP" gcp-sev-${{ inputs.distrib }}.pkr.hcl
+
+      # - name: Build GCP images - release
+      #   if: startsWith(github.ref, 'refs/tags')
+      #   run: |
+      #     packer build -var "prefix=release-${GITHUB_REF_NAME//./-}" gcp-sev-${{ inputs.distrib }}.pkr.hcl  
+
+  test-image:
+    name: Test image
+    runs-on: ubuntu-22.04
+    needs: build-kms-cosmian-vm-image
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: GCP auth
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: "${{ secrets.GOOGLE_COSMIAN_DEV_CREDENTIALS }}"
+
+      - name: Set up Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+        with:
+          version: latest
+          install_components: beta
+
+      - name: Launch GCP instance
+        id: run-gcp-instance
+        env:
+          IMAGE_NAME: ${{ needs.build-kms-cosmian-vm-image.outputs.image_name }}
+          CI_INSTANCE: ${{ needs.build-kms-cosmian-vm-image.outputs.ci_instance }}
+        run: |
+          gcloud beta compute instances create $CI_INSTANCE --machine-type n2d-standard-2 \
+                                                            --zone europe-west4-a \
+                                                            --min-cpu-platform="AMD Milan" \
+                                                            --confidential-compute-type=SEV_SNP \
+                                                            --maintenance-policy=TERMINATE \
+                                                            --image="$IMAGE_NAME" \
+                                                            --image-project=$GCP_DEV_PROJECT \
+                                                            --project $GCP_DEV_PROJECT \
+                                                            --tags ssh-full,backend,http-server,https-server,cosmian-vm-agent \
+                                                            --metadata-from-file=startup-script=scripts/gcp-start.sh \
+                                                            --max-run-duration=10m \
+                                                            --instance-termination-action=DELETE
+          IP_ADDR=$(gcloud beta compute instances describe $CI_INSTANCE --format='get(networkInterfaces[0].accessConfigs[0].natIP)' --zone=europe-west4-a)
+          echo "IP_ADDR=${IP_ADDR}" >> "$GITHUB_OUTPUT"
+
+      - name: Download Cosmian VM client
+        run: |
+          wget https://package.cosmian.com/cosmian_vm/1.0.1/cosmian_vm
```

Comment:
should we use same version as `cosmian-vm-1-1-0-rc-2-sev-rhel`?

Reply:
indeed
```suggestion
          wget https://package.cosmian.com/cosmian_vm/1.1.0-rc.2/cosmian_vm
```